packages/rich-text/src/component/README.md

```diff
@@ -0,0 +1,53 @@
+# `RichText`
+
+Render a rich [`contenteditable` input](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Editable_content), providing users with the option to format the content.
```

Comment:
In the sprit of moving towards a more cross-platform RichText, I think we should avoid referencing `content editable` in the description as that will only be accurate in the web implementation.

Reply:
Sure, sorry, was just moving files. :)

Reply:
You could explain that it's cross-platform and it happens to be `contenteditable` on web and Aztec in React Native context :)